api/billingd/service/service.go

```diff
@@ -952,8 +953,11 @@ func (s *Service) handleUsage(ctx context.Context, cus *Customer, product Produc
 			product.Key + "_usage":      total,
 			product.Key + "_units":      product.Units,
 			product.Key + "_free_quota": product.FreeQuotaSize,
+			"grace_period_start":        s.analytics.FormatUnix(cus.GracePeriodStart),
+			"invoice_period_end":        s.analytics.FormatUnix(cus.InvoicePeriod.UnixEnd),
+			"invoice_period_start":      s.analytics.FormatUnix(cus.InvoicePeriod.UnixStart),
+			"subscription_status":       cus.SubscriptionStatus,
 		}
-		go s.analytics.Update(cus.Key, cus.Email, false, summary)
```

Comment:
not updating here as it will be updated daily + captured in the grace period start/end events below